app/assets/stylesheets/components/common/_lists.scss

```diff
@@ -93,7 +93,26 @@
 // Default list style for editorial content
 .editorial ul {
   @extend .list;
+
+  // Override for editorial unstyled lists
+  &.unstyled-list:before {
+    content: none;
+  }
+}
+
+/*
+.editorial .unstyled-list {
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+
+  > li {
+    &:before {
+      content: none;
+    }
+  }
 }
+*/
```

Comment:
I think I added it then commented it out(!). Should have deleted it before pushing but didn't. My fault. Happy to delete or will you do that?